src/maxdiffusion/max_utils.py

```diff
@@ -150,27 +150,6 @@ def add_text_to_summary_writer(key, value, summary_writer):
     summary_writer.add_text(key, value)
 
 
-def write_metrics_for_gcs(metrics, step, config, running_metrics):
```

Comment:
@sizhit2 Why are we removing this ?

Reply:
Because there is a function with same function signature defined in line 120. I don't think we will ever call this one...